docker-compose.yml

```diff
@@ -87,6 +87,7 @@ services:
       - ./docker/message_broker/certs:/broker-certs:ro,z
       - request-logs-volume:/var/log/iib/requests:z
       - request-related-bundles-volume:/var/lib/requests/related_bundles:z
+      - request-recursive-related-bundles-volume:/var/lib/requests/recursive_related_bundles:z
```

Comment:
Are you planning on getting this merged before or after https://github.com/release-engineering/iib/pull/421? The latter one will need to be updated after merging the former.

Reply:
yes, I am since this one is a bigger change